Linearsearch.py

```diff
@@ -0,0 +1,9 @@
+#Linear search algorithm in python 
+def search(array, key): 
+  
+    for i in range(len(array)): 
+  

```

Comment:
Delete the Extra space on line 5 and line 8

Reply:
i have updated the code once check please